pkg/services/serviceaccounts/serviceaccounts.go

```diff
@@ -7,13 +7,29 @@ import (
 	"github.com/grafana/grafana/pkg/services/user"
 )
 
-// this should reflect the api
+/*
+ServiceAccountService is the service that manages service accounts.
+
+Service accounts are used to authenticate API requests. They are not users and
+do not have a password.
+*/
 type Service interface {
 	CreateServiceAccount(ctx context.Context, orgID int64, saForm *CreateServiceAccountForm) (*ServiceAccountDTO, error)
 	DeleteServiceAccount(ctx context.Context, orgID, serviceAccountID int64) error
 	RetrieveServiceAccountIdByName(ctx context.Context, orgID int64, name string) (int64, error)
 }
 
+/*
```

Comment:
comments outside of the scope of this PR. this comment reflects on the migration of apikeys to service accounts